go.mod

```diff
@@ -49,6 +49,7 @@ require (
 	github.com/kelseyhightower/envconfig v1.4.0
 	github.com/lonelycode/go-uuid v0.0.0-20141202165402-ed3ca8a15a93
 	github.com/lonelycode/osin v0.0.0-20160423095202-da239c9dacb6
+	github.com/magiconair/properties v1.8.1
```

Comment:
Please run `go mod tidy` I think this was added by accident it is not used anywhere.

Reply:
done!